app/services/hyrax/adapters/nesting_index_adapter.rb

```diff
@@ -33,19 +38,22 @@ def self.find_preservation_parent_ids_for(id:)
         end
       end
 
+      ##
       # @api public
+      #
       # @param id [String]
       # @return Samvera::NestingIndexer::Documents::IndexDocument
       def self.find_index_document_by(id:)
         solr_document = find_solr_document_by(id: id)
         coerce_solr_document_to_index_document(original_solr_document: solr_document, id: id)
       end
 
+      ##
       # @api public
       # @yieldparam id [String]
       # @yieldparam parent_id [Array<String>]
-      # Samvera::NestingIndexer.reindex_all!(extent: FULL_REINDEX)
-      # rubocop:disable Lint/UnusedMethodArgument
```

Comment:
You need to keep this method in place;

The interface specs test if this method takes a block.  The way to test that interface is through parameters (e.g. `&block`)  However it doesn't use `block.call` (The original author, likely me, must've chosen `yield`)

Reply:
sorry i pushed the patch for this some time ago, but forgot to readd the rubocop.

we should probably fix the upstream test `&block` has historically been much slower than omitting it. it should be fine to test the actual `yield` in the shared examples, right?

Reply:
The shared examples lean on method signatures/parameters.  So yes, we could change that test, however, it'll be much more expensive.